torchmetrics/text/rouge.py

```diff
@@ -111,11 +117,9 @@ def __init__(
                 raise ValueError(f"Got unknown rouge key {key}. Expected to be one of {ALLOWED_ROUGE_KEYS}")
 
         self.rouge_keys = rouge_keys
-        self.newline_sep = newline_sep
-        self.use_stemmer = use_stemmer
-        self.aggregator = BootstrapAggregator()
-        self.scorer = RougeScorer(rouge_keys, use_stemmer=self.use_stemmer)
-        self.decimal_places = decimal_places
+        self.rouge_keys_values = [ALLOWED_ROUGE_KEYS[key] for key in rouge_keys]
+        self.stemmer = nltk.stem.porter.PorterStemmer() if use_stemmer else None
```

Comment:
is this the only part nltk is used? If so can we maybe make the import conditional of `use_stemmer`?

Reply:
Actually, there are 2 places where the `nltk` package:
1. when `use_stemmer=True`
2. when `RougeLsum` is calculated.

Though, there are still plenty of cases where `nltk` is not required. Using `import nltk` conditionally is a good idea.